newsletter_service.py

```diff
@@ -19,88 +19,148 @@
 Response:
   - newsletter (string): formatted newsletter with recommendation and all article summaries
 """
+# Getting all personas and topics stored in settings document in the newsletter_components collection
+get_persona = db_service.get_components_from_firestore('gcp_newsletter', 'settings')['persona']
+get_topic = db_service.get_components_from_firestore('gcp_newsletter', 'settings')['topic']
 
+# Create a matrix to iterate through every combination of persona and topics
+persona_topic_matrix = [(p, t) for p in get_persona for t in get_topic]
 
-def get_newsletter_from_sources(source="https://snownews.appspot.com/feed",
-                                time_period="day",
-                                user_topic="Any",
-                                user_persona="All"):
+def get_newsletter_from_sources(source="https://snownews.appspot.com/feed"):
```

Comment:
Please confirm: get_newsletter_from_sources() is going to be the function run by our daily batch job? That is why we remove the time_period parameter - because we know it will be run daily. If yes, we may want to reconsider the function name. 

Also it might be worth having time_period parameter even for the daily batch job in case we every miss days or need to retroactively fill certain days. I think we can worry about that at a future date. TODO.

Reply:
Yes, I removed due to running daily. I can add a time_period. I created a utility script to back fill data, but makes sense to parameterize that. Instead of being week or day, maybe just a number value with the number of days back we want to fill.

Reply:
Added Parameter as suggested.